meinberlin/apps/contrib/assets/CheckboxBtn.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 
-export default class SwitchButton extends React.Component {
+export default class CheckboxBtn extends React.Component {
```

Comment:
I cannot find the CheckboxBtn used anywhere, but also not the SwitchButton and am slightly confused